chirp/drivers/tdh8.py

```diff
@@ -2093,11 +2078,12 @@ class TDH8_GMRS(TDH8):
 
     def validate_memory(self, mem):
         msgs = super().validate_memory(mem)
-        if mem.freq / 1000000 in GMRS_FREQS3:
-            pass
+        if mem.freq not in GMRS_FREQS:
+            msgs.append(chirp_common.ValidationError(
+                "The frequency in channels 31-55 must be between"
```

Comment:
This error will always be given for any non-GMRS frequency, even for channels 1-30 and >55 because `validate_memory()` is called before `set_memory()` by the GUI. Are you actually testing this code locally with the GUI?